src/backends/ilp-quoter/index.js

```diff
@@ -0,0 +1,105 @@
+'use strict'
+
+const _ = require('lodash')
+const request = require('co-request')
+const BigNumber = require('bignumber.js')
+const AssetsNotTradedError = 
+  require('../../errors/assets-not-traded-error')
+const NoAmountSpecifiedError = 
+  require('../../errors/no-amount-specified-error')
+const UnacceptableQuoterAmountError = 
+  require('../../errors/unacceptable-quoter-amount-error')
+const log = require('../../services/log')('ilpquote')
+const config = require('../../services/config')
+
+function lookupCurrencies (source_ledger, destination_ledger) {
+  for (let pair of config.get('tradingPairs').toJS()) {
+    if (pair[0].indexOf(source_ledger) === 4 &&
```

Comment:
Maybe add a comment to the code for clarity? I find it hard to follow without the explanation. Also, I would argue for naming the currency variables:
```
const currencyA = pair[0].slice(0, 3)
const currencyB = pair[1].slice(0, 3)
return [currencyA, currencyB]
```
or something like that. Otherwise it's not clear to me what this method returns.